src/test/java/org/folio/edge/rtac/MainVerticleTest.java

```diff
@@ -153,79 +166,102 @@ public void testAdminHealth(TestContext context) {
     assertEquals("\"OK\"", resp.body().asString());
   }
 
+  // unsuccessful login attempts result in a 200 OK status with empty elements 
+  // in the message body
+
   @Test
   public void testRtacUnknownApiKey(TestContext context) throws Exception {
     logger.info("=== Test request with unknown apiKey (tenant) ===");
 
     final Response resp = RestAssured
+      .given()
+      .accept(APPLICATION_JSON)
       .get(String.format("/prod/rtac/folioRTAC?mms_id=%s&apikey=%s", titleId, unknownTenantApiKey))
       .then()
-      .contentType(APPLICATION_XML)
+      .contentType(APPLICATION_JSON)
       .statusCode(200)
-      .header(HttpHeaders.CONTENT_TYPE, APPLICATION_XML)
+      .header(HttpHeaders.CONTENT_TYPE, APPLICATION_JSON)
       .extract()
       .response();
 
-    String expected = new Instances().toXml();
-    String actual = resp.body().asString();
-    assertEquals(expected, actual);
+    JsonObject result =  new JsonObject(resp.body().asString());
+
+    assertNull(result.getString("instanceId"));
+    assertEquals(0, result.getJsonArray("holdings").size());
```

Comment:
I think it is worth extracting a method to make it obvious that this empty result is intended to be a failure.

Similarly, I think the name of the test could be changed to make it apparent this is a failure scenario.

I made a start on doing something like that in this [branch](https://github.com/folio-org/edge-rtac/tree/EDGRTAC-64-explicit-expectations). That could act as an example.